providers/agent/mcorpc/audit/choria_test_unix.go

```diff
@@ -0,0 +1,27 @@
+// Copyright (c) 2020-2021, R.I. Pienaar and the Choria Project contributors
+//
+// SPDX-License-Identifier: Apache-2.0
+//go:build !windows
+// +build !windows
+
+package audit
+
+import (
+	"os"
+	"os/user"
+	"strconv"
+	"syscall"
+
+	"github.com/onsi/gomega"
```

Comment:
Couldn't use dot import due to staticcheck:

```
providers/agent/mcorpc/audit/choria_test_unix.go:15:2: should not use dot imports (ST1001)
```

Reply:
Thats pretty weird, I'll see if I can fix it post merge, probably staticheck doesnt figure this is a test file